website/content/api-docs/system/config-ui-login-default-auth.mdx

```diff
@@ -0,0 +1,180 @@
+---
+layout: api
+page_title: /sys/config/ui/login/default-auth - HTTP API
+description: The '/sys/config/ui/login/default-auth' endpoint configures default authentication types that will display on a UI page per namespace.
+---
+
+# `/sys/config/ui/login/default-auth`
+
+@include 'alerts/enterprise-only.mdx'
+
+@include 'alerts/restricted-admin.mdx'
+
+The `/sys/config/ui/login/default-auth` endpoint is used to list, create, read, update, or delete login customization configurations for the Vault GUI. These configurations determine which methods to display in the login form.
+
+## Create UI default login customization configuration
+
+This endpoint allows the creation of a new UI default login customization configuration. Only one UI default login configuration is
```

Comment:
I think `default` can be removed here since that's implied by the title and first sentence?
```suggestion
This endpoint allows the creation of a new UI default login customization configuration. Only one UI login configuration is
```